framework/caching/Cache.php

```diff
@@ -202,14 +210,19 @@ public function multiGet($keys)
      * @param mixed $key a key identifying the value to be cached. This can be a simple string or
      * a complex data structure consisting of factors representing the key.
      * @param mixed $value the value to be cached
-     * @param int $duration the number of seconds in which the cached value will expire. 0 means never expire.
+     * @param int $duration the number of seconds in which the cached value will expire. If not set - the value of [[ttl]] is used,
```

Comment:
@param int default duration in seconds before the cache will expire. Default value is 0, meaning infinity.